source/_posts/2024-10-02-php-core-roundup-19.md

```diff
@@ -0,0 +1,154 @@
+
+Maintaining a mature, reliable, and widely used programming language that continues to get new features and improvements for almost 30 years involves a lot of work! Maintaining PHP documentation, php.net infrastructure, translations, triaging issues and security reports, bug fixes, mailing lists, reviewing RFCs, and packaging PHP only to name a few, we have PHP contributors and PHP Foundation members putting a commendable amount of effort into improving PHP!
+
+While these PHP Core Roundup series may not write about these contributions often because we want to keep these posts exciting and resourceful, we want to spread love to all the contributors to the PHP ecosystem 💜.
+
+
+## PHP Release Cycle Updates
+
+In April, we voted and accepted an [RFC](https://wiki.php.net/rfc/release_cycle_update) to update our Release Cycle policy.
+
+The PHP Core team provided two years of active support, followed by only one year of security fixes. We now have a new release cycle that, from PHP 8.1 (released in 2021 November), all PHP versions now get **two years of security fixes instead of one year**. The two-year active support period remains the same.
+
+Further, we have changed the active-support and End-Of-Life dates to align to December 31st of the calendar year. This makes the support and EOL dates more predictable.
+
+Here are the updated active-support and EOL dates for the current line-up of PHP versions. Dates changed from the previously set dates are in bold.
+
+
+|PHP version|Release date|Active maintenance until|EOL date|
+|:--:|:--:|:--:|:--:|
+|PHP 8.1|2021-11-25|2023-11-25|**2025-12-31**|
+|PHP 8.2|2022-12-08|**2024-12-31**|**2026-12-31**|
+|PHP 8.3|2023-11-23|**2025-12-31**|**2027-12-31**|
+|PHP 8.4|2024-11-21|2026-12-31|2028-12-31|
+|PHP 8.5|2025-11|2027-12-31|2029-12-31|
+
+
+## PHP Core Developments
+
+A few days ago, PHP 8.4 reached its feature-freeze. PHP 8.4 is slated to be released on November 21st of this year.
+
+The first release candidate for PHP 8.4 is already out — they are not production-ready, but they are available to try out and test PHP application on, either by compiling from [source](https://github.com/php/php-src/tags), using [Windows binaries](https://windows.php.net/qa/), or in [OCI containers](https://hub.docker.com/_/php/tags?name=8.4-).
+
+### Highlights for PHP 8.4
+
+PHP 8.4 is an important release that brings major new features, several updates to the build dependencies and underlying libraries, and a fair bit of deprecations to iron out some of the legacy and undesirable behaviors and features in legacy PHP versions.
+
+Using various approaches, each PHP version brings a lot of performance improvements and security tightening too. In PHP 8.4, we continue this in this direction with several minor internal improvements as well as noticeable improvements in JIT, and PHP extensions such as mbstring, BCMath, XML extensions, PCRE, and more.
+
+Further, PHP 8.4 moves IMAP, Pspell, OCI, and pdo_oci extensions to PECL.
```

Comment:
```suggestion
Further, PHP 8.4 unbundles IMAP, Pspell, OCI, and pdo_oci extensions. It means if you wanted to continue using them, you'll have to install via PECL.
```

Reply:
Thank you, I think this makes it more helpful.